$OpenBSD: patch-configure,v 1.2 2010/03/21 10:41:37 ajacoutot Exp $

deskbar-applet is not really needed for tracker to build but we still
want the functionnality. We cannot add deskbar-applet as a build
dependency without creating a loop:
evince -> nautilus -> tracker -> deskbar-applet -> py-gnome-desktop -> evince

--- configure.orig	Thu Feb  4 11:56:45 2010
+++ configure	Thu Feb  4 14:09:19 2010
@@ -16007,17 +16007,6 @@ else
   enable_deskbar_applet=auto
 fi
 
-
-if test "x$enable_deskbar_applet" != "xno" ; then
-   if pkg-config --atleast-version=2.19 deskbar-applet ; then
-      have_deskbar_applet="module"
-   elif pkg-config --atleast-version=2.16 deskbar-applet ; then
-      have_deskbar_applet="handler"
-   else
-      have_deskbar_applet="no"
-   fi
-fi
-
 if test "x$enable_deskbar_applet" = "xyes"; then
    if test "x$have_deskbar_applet" = "xno"; then
       as_fn_error "Couldn't find deskbar-applet >= 2.16." "$LINENO" 5
